build-system/tasks/extension-helpers.js

```diff
@@ -615,8 +616,8 @@ function buildBinaries(extDir, binaries, options) {
       entryPoint,
       `${extDir}/dist`,
       Object.assign(options, {
-        toName: maybeToEsmName(`${name}.max.js`),
-        minifiedName: maybeToEsmName(`${name}.js`),
+        toName: npmMaybeToEsmName(`${name}.max.js`),
+        minifiedName: npmMaybeToEsmName(`${name}.js`),
```

Comment:
FMI: Why won't this give the npm naming to regular extension builds? It looks like `buildBinaries` is used here as well, where we would want the regular esm name:
https://github.com/ampproject/amphtml/blob/7aeb64c073d68050e5ca0ed9bd4e7820ed8737c2/build-system/tasks/extension-helpers.js#L491

Reply:
`buildBinaries` is only used for generating NPM binaries